src/main/java/org/spongepowered/api/data/manipulator/mutable/tileentity/FurnaceData.java

```diff
@@ -36,36 +35,37 @@
 public interface FurnaceData extends DataManipulator<FurnaceData, ImmutableFurnaceData> {
 
     /**
-     * Gets the {@link MutableBoundedValue} for the remaining burn time of the
-     * {@link Furnace}.
+     * Gets the {@link MutableBoundedValue} for the already passed burn time of the {@link Furnace}.
+     * When the {@link #passedBurnTime()} is equals the {@link #maxBurnTime()} the current used fuel
+     * is depleted.
      *
-     * @return The value for the remaining burn time
+     * @return The value for the already passed burn time
      */
-    MutableBoundedValue<Integer> remainingBurnTime();
+    MutableBoundedValue<Integer> passedBurnTime();
 
     /**
-     * Gets the {@link MutableBoundedValue} for the remaining cook time of the
-     * {@link Furnace}.
+     * Gets the {@link MutableBoundedValue} for the time the current fuel in the {@link Furnace} can
+     * burn totally.
```

Comment:
This should probably say something like 'the maximum amount of time the furnace can burn for.'